### ISSUE TITLE
[Bug] `migrate up --verbose 1` shows statements for all unapplied migrations

### ISSUE THREAD

**Issue by sameoldmadness** [prisma/migrate#205]
Given that I have two new migrations
And I run `prisma migrate up --verbose 1`
I see SQL statements for both migrations

```
SQL Commands:

-- Start Migrations

-- Migration 20191028233001-first
...

-- Migration 20191029113029-second
...

-- End Migrations
```

**Comment by sameoldmadness** [comment]
In `prisma2 lift up 1 --verbose` "verbose" flag would be ignored. Not sure if this is expected.

**Comment by pantharshit00** [comment]
Unable to reproduce this in the latest version. I don't see the SQL statements of the last migration.

**Comment by tomhoule** [comment]
Thanks for taking the time to report this issue!

We've recently released a Prisma Migrate Preview ([2.1.3.0 release notes](https://github.com/prisma/prisma/releases/tag/2.13.0)), which has quite a few changes compared to the previous experimental version. We believe this issue is no longer relevant in this new version, so we are closing this.

We would encourage you to try out Prisma Migrate Preview. If you come across this or any other issue in the preview version, please feel free to open a [new issue in prisma/prisma](https://github.com/prisma/prisma/issues/new/choose).
For general feedback on Prisma Migrate Preview, feel free to chime in [on this issue](https://github.com/prisma/prisma/issues/4531).
